color_refinement.py

```diff
@@ -103,6 +119,8 @@ def get_number_isomorphisms(g: Graph, h: Graph, count: bool) -> int:
     if len(g.edges) != len(h.edges):
         return 0
     added_graph = g + h
+    preprocessing.remove_loners(added_graph)
+    preprocessing.checks(g, h)
```

Comment:
Now you don't use your pretty check :(
Idea:
`if not preprocessing.check(g,h):
    return 0`